core/pkg/util/timeutil/timeutil.go

```diff
@@ -128,6 +128,8 @@ func FormatStoreResolution(dur time.Duration) string {
 	}
 	if dur >= time.Hour {
 		return fmt.Sprintf("%dh", int(dur.Hours()))
+	} else if dur >= time.Minute {
```

Comment:
What is the purpose of this else if? I already see a 10 min duration condition below .. SHouldnt that be the one that creates the string 10m?